osmosis-1/osmosis.zone_assets.json

```diff
@@ -2314,6 +2314,13 @@
       "path": "transfer/channel-1575/aISLM",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "pundix",
+      "base_denom": "bsc0x29a63F4B209C29B4DC47f06FFA896F32667DAD2C",
```

Comment:
Checks are failing because bsc0x29a63F4B209C29B4DC47f06FFA896F32667DAD2C is not registered to the Chain Registry. I think the issue here is the inconsistent capitalization, which indeed matters for identifying denoms.
The Chain Registry has bsc0x29a63f4b209c29b4dc47f06ffa896f32667dad2c, and correcting these two lines to this denom should resolve the issue,

Reply:
Thank you for the reminder; we will unify the base denom.

Reply:
Chain Registry(cosmos/chain-registry#3496) has been modified and is awaiting merging.